README.md

```diff
@@ -105,6 +105,9 @@ The following label has to be added to the Machine spec within the MachineSet sp
 
 Not having these labels will result in the Windows node not being marked as a worker.
 
+If the Machine spec has the label `windowsmachineconfig.openshift.io/ignore=true`, the Machine will be ignored by WMCO,
```

Comment:
should this information be moved to the hacking.md file? Wondering if this would be useful in the README if we don't expect users to use this other than for development and testing.

Reply:
That's true, I was back and forth about this. I can move it there